Xamarin.Essentials/Calendars/Calendars.shared.cs

```diff
@@ -47,6 +47,24 @@ public static async Task<CalendarEvent> GetEventAsync(string eventId)
             return await PlatformGetEventAsync(eventId);
         }
 
+        public static Task<CalendarEvent> GetEventInstanceAsync(string eventId, DateTimeOffset instanceDate) => PlatformGetEventInstanceAsync(eventId, instanceDate);
+
+        public static Task<string> CreateCalendarEvent(CalendarEvent newEvent) => PlatformCreateCalendarEvent(newEvent);
+
+        public static Task<bool> UpdateCalendarEvent(CalendarEvent eventToUpdate) => PlatformUpdateCalendarEvent(eventToUpdate);
+
+        public static Task<bool> SetEventRecurrenceEndDate(string eventId, DateTimeOffset recurrenceEndDate) => PlatformSetEventRecurrenceEndDate(eventId, recurrenceEndDate);
+
+        public static Task<bool> DeleteCalendarEventInstanceByDate(string eventId, string calendarId, DateTimeOffset dateOfInstanceUtc) => PlatformDeleteCalendarEventInstanceByDate(eventId, calendarId, dateOfInstanceUtc);
```

Comment:
To fit with more .NET-y things, maybe a better name would be `DeleteEventInstanceAsync` and then the parameters can overload to be the ID and the instance date. This way, the popup in intellisense groups related actions/object together.